spec/system/planning_applications/reviewing_spec.rb

```diff
@@ -297,7 +297,7 @@
           click_link "Publish the recommendation"
           click_button "Determine application"
 
-          expect(page).to have_content("The Decision Notice cannot be sent. Please try again later.")
+          expect(page).to have_content("The email cannot be sent. Please try again later.")
```

Comment:
Should we also have a feature spec that checks we send the email when validating?

Reply:
I've added a line into the validation spec to verify that the delivered mails count increments by 1 on validation, but if you can think of a more complete way to do it, please let me know. I figured we didn't need to validate the content as we are already doing this in the mailer spec.